x-pack/plugins/telemetry_collection_xpack/schema/xpack_plugins.json

```diff
@@ -4044,6 +4044,29 @@
             },
             "total": {
               "type": "long"
+            },
+            "deprecated": {
+              "type": "long"
+            },
+            "app": {
```

Comment:
Fixme: the `app` and `layout` properties should only be part of the `printable_pdf` type

This was unintentional from cleaning up an `any` in the code.